sdk/storage/azure-storage-queue/azure/storage/queue/queue_client.py

```diff
@@ -110,8 +110,14 @@ def __init__(
         self._query_str, credential = self._format_query_string(sas_token, credential)
         super(QueueClient, self).__init__(parsed_url, service='queue', credential=credential, **kwargs)
 
-        self._config.message_encode_policy = kwargs.get('message_encode_policy') or TextXMLEncodePolicy()
-        self._config.message_decode_policy = kwargs.get('message_decode_policy') or TextXMLDecodePolicy()
+        if 'message_encode_policy' in kwargs:
+            self._config.message_encode_policy = kwargs['message_encode_policy'] or NoEncodePolicy()
+        else:
+            self._config.message_encode_policy = TextXMLEncodePolicy()
+        if 'message_decode_policy' in kwargs:
+            self._config.message_decode_policy = kwargs['message_decode_policy'] or NoDecodePolicy()
```

Comment:
Should we document these kwargs in the client docstring or should they be part of the configuration kwargs in #6165 ?

Reply:
They should be documented in the Client I think - as they are not part of the pipeline and not common between the storage library. I will add this :)

Reply:
Actually, correction - after the meeting today it seems that the other languages do not currently support this feature, therefore we will not be documenting it right now.